src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -235,6 +238,21 @@ public void setDrivePowers(double xPower) {
         setDrivePowers(xPower, 0.0, 0.0, true);
     }
 
+    private ProfiledPIDController thetaController = new ProfiledPIDController(
+        0.5, 0, 0, 
+        new TrapezoidProfile.Constraints(
+            SwerveSubsystem.MAX_OMEGA / 2.,
```

Comment:
Not super important for R2, but this can maybe use the constants already in the subsystem so that it also works for 2020?